src/style.css

```diff
@@ -1,3 +1,94 @@
-body {
+
+html {
+  font-family: Roboto, sans-serif;
+
+  --blue-link:#00ACDC;
+}
+
+.body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  margin: 0;
+  padding: 0 50px;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.header__nav {
+  display: block;
+}
+
+.header__nav-list {
+  padding: 0;
   margin: 0;
+
+  display: flex;
+  list-style: none;
+}
+
+.header__nav-item {
+  padding-right: 20px;
+}
+
+.header__nav-link {
+  box-sizing: content-box;
+
+  display: flex;
+  align-items: center;
+
+  height: 60px;
+
+  text-decoration: none;
+  text-transform: uppercase;
+  text-align: left;
+
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+
+  position: relative;
+
+  transition: color 0.3s;
+}
+
+.header__nav-item .header__nav-link:hover {
+  color: var(--blue-link);
+}
```

Comment:
There is no need for this relationship between nav-item and nav-link, apply style to nav-link only